docs/UserGuide.md

```diff
@@ -332,35 +343,39 @@ Examples:
 
 ### Listing all events a volunteer is participating in: `/v view`
 
-Displays all events the volunteer at the specified `INDEX` under the `Volunteers` display is participating in.<br/>
+Want to see which events a volunteer is joining? This command shows you everything they're signed up for.
+It will display all events the volunteer at the specified `INDEX` under the `Volunteers` display is participating in.<br/>
 
 Format: `/v view VOLUNTEER_INDEX`
 
-* The index refers to the number on the left of the volunteer's name in the Volunteer list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The `INDEX` is the number you see next to their name in the list
 
 <div markdown="span" class="alert alert-primary">:bulb: **Note:**
-you can use <a href="#listing-all-volunteers-and-events--list">`list`</a> command to show all volunteers and events again!</div>
+After you're done, type <a href="#listing-all-volunteers-and-events--list">`list`</a> to see all events again!</div>
 
 Examples:
 
+* `/v view 1` displays the events the first volunteer is participating in.
+
 ![View Volunteer Demo](images/commandDemo/ViewVolunteerDemo.png)
 
-* `/v view 1` displays the events the first volunteer is participating in.
 
 [Back To Top :arrow_heading_up:](#learn-about-commands-supported-by-volunsync)
 
+--------------------------------------------------------------------------------------------------------------------
+
 <div style="page-break-after: always;"></div>
 
 ## Event-related Commands
 
 ### Adding an event: `/e new`
 
-Adds an event to the database.
+Planning a new activity? Here's how to add it to VolunSync.<br>
+It will add an event to the database with the specified details.
 
 Format: `/e new n/EVENT_NAME l/LOCATION d/DATE s/START_TIME e/END_TIME [des/DESCRIPTION]`
 
-Input Requirements:
+What you'll need:
 - Event Name: Alphanumeric characters and spaces only, 1-50 characters.
 - Location: Alphanumeric characters and spaces only, 1-100 characters.
 - Date: Format YYYY-MM-DD (e.g. 2024-01-01),
```

Comment:
Maybe add 24-hour in-front of format for end time too

Reply:
resolved